apps/portal/lib/graphql/services/Firebase.service.ts

```diff
@@ -0,0 +1,49 @@
+import { PrismaClient } from '@prisma/client';
+import { singleton } from 'tsyringe';
+const { initializeApp, getApps, applicationDefault, cert } = require('firebase-admin/app');
+const { getFirestore, Timestamp, FieldValue } = require('firebase-admin/firestore');
+const serviceAccount = require('C:/UTD/Senior Year/Spring Semester 2022/Projects/try-acm/practice-5d375-firebase-adminsdk-tbfef-003353b527.json');
+require('dotenv').config();
+
+@singleton()
+export default class Firebase {
```

Comment:
potentially nit: Consider changing the name from `Firebase` to `FirebaseService`, just in case the keyword `Firebase` conflicts with something in `firebase-admin`.

Reply:
Ok